packages/siwe/exports/index.ts

```diff
@@ -1,8 +1,25 @@
-import type { SIWEConfig, SIWESession } from '@web3modal/core'
+import type {
+  SIWEConfig,
+  SIWESession,
+  SIWECreateMessageArgs,
+  SIWEVerifyMessageArgs,
+  SIWEClientMethods
+} from '../core/utils/TypeUtils.js'
```

Comment:
can't you do directly `export type { ... } from '../core/utils/TypeUtils.js`?

Reply:
Some are from different origin and some are used in the module so this would add extra lines of code + repeated path declarations